doc_user/UserManual.adoc

```diff
@@ -630,7 +631,26 @@ to the expiring date. If the first selected lot does not contain a high enough q
 movement, several discharging movements may be generated. Ask the Administrator or check the _Administrator’s Guide_
 for more information.
 
-<<<
+[#delete-last-stock-movement]
+===== 4.2.2.3 Delete Last Stock Movement (*[.underline]##D##elete*)
+To delete the last movement, select first the row that you want to delete and press the *[.underline]##D##elete* button in the *_Stock Movement Browser_* window. if it is not the last movement you will have and *_Warning_* window:
+
+image:image250.png[Warning]
```

Comment:
This should be an ERROR and not just a warning; see dialog title.

Reply:
the title is Warning

Reply:
I think so as a warning to me means something to know about but we will continue anyway.   Doesn't this really mean you can't do it?  So it should be IMHO an error.

Reply:
okay you right but we already merge the gui so can I create the task to update the window and use error dialog instead of warning dialog?

Reply:
It is actually just warning even before the operation takes place, so I think it is different from an ERROR which comes during an operation, but I think the point here is consistency with the rest of the doc.

According with the PR merged in GUI it is a Warning, maybe we can have it consistent with the GUI as for now and change it later on?

Reply:
I've checked, and according with the rest of the code and documentation, we used warning only few times and for not-blocking operations, we used error for all other cases instead.

So @ArnaudFonzam you can include this change in the new PR on GUI

Reply:
Okay

Reply:
and update this doc accordingly, thanks!

Reply:
okay I'm working on it